src/Native/configure.cmake

```diff
@@ -171,6 +171,36 @@ endif ()
 
 check_cxx_source_runs(
     "
+    #include <stdlib.h>
+    #include <time.h>
+    #include <sys/time.h>
+    int main()
+    {
+        int ret; 
+        struct timespec ts;
+        ret = clock_gettime(CLOCK_MONOTONIC, &ts);
+        exit(ret);
+    }
+    " 
+    HAVE_CLOCK_MONOTONIC)
+
+check_cxx_source_runs(
+    "
+    #include <stdlib.h>
+    #include <mach/mach_time.h>
+    int main()
+    {
+        int ret;
+        mach_timebase_info_data_t timebaseInfo;
+        ret = mach_timebase_info(&timebaseInfo);
+        mach_absolute_time();
+        exit(ret);
+    }
+    " 
+    HAVE_MACH_ABSOLUTE_TIME)
+
+check_cxx_source_runs(
+    "
```

Comment:
The HAVE_MACH_ABSOLUTE_TIME should probably be _compiles rather than _runs.  I'll change that.  I'm not sure about HAVE_CLOCK_MONOTONIC, as the docs seem to imply that CLOCK_MONOTIC could be defined but then result in clock_gettime failing due to it not being supported.